tools/topology/topology2/cavs-nocodec.conf

```diff
@@ -988,16 +988,14 @@ IncludeByKey.PASSTHROUGH {
 					direction	"playback"
 					name "SSP2 Playback"
 					formats 'S16_LE,S24_LE,S32_LE'
-					rate_min 8000
-					rate_max 192000
+					rates "8000,11025,16000,22050,32000,44100,48000,64000,88200,96000,176400,192000"
```

Comment:
missing 24000?

Reply:
Yep. I'll test it if can be used.

Reply:
Seems it can't be used. In tplg build:

```
[1/142] Generating sof-tgl-nocodec.tplg
saving nhlt as binary in nhlt-sof-tgl-nocodec.bin
ALSA lib pcm.c:366:(split_rate) unsupported stream rate 24000
ALSA lib parser.c:211:(tplg_load_config) failed to parse topology
Unable to load configuration: Invalid argument

```

Reply:
Yes, I think ALSA does not define this  24kHz rate by default but it should be fine if we used a KNOT approach - which may mean a driver change. @ujfalusi @ranj063 FYI.